### ISSUE TITLE
Dependency "EVReflection" has no shared framework schemes

### ISSUE THREAD

**Issue by mhillebrand** [evermeer/EVReflection#151]
$ carthage update
*** Fetching EVReflection
*** Checking out EVReflection at "4.0.10"
*** xcodebuild output can be found in /var/folders/ml/d9hzjcrd3mv_3hj_3qqf__brkdn63y/T/carthage-xcodebuild.DpmGqP.log
*** Skipped building EVReflection due to the error:
Dependency "EVReflection" has no shared framework schemes

**Comment by mhillebrand** [comment]
This may be helpful: http://stackoverflow.com/questions/35054788/carthage-no-shared-framework-schemes-for-ios-platform-for-my-own-framework

**Comment by evermeer** [comment]
Thanks for reporting this. I will hava a look at this tomorrow.

**Comment by evermeer** [comment]
Hmm, I moved the shared frameworks for Carthage to it's own project named Carthage. 
For some reason it wasn't in the workspace anymore. I just added it back.
I have never worked with Carthage so I hope it works.
Please let me know if it's still not OK.

You do have to be aware that there are separate targets for each platform.
So you have EVReflection_iOS, EVReflection_OSX, EVReflection_WatchOs and EVReflection_TVOS.

Those targets currently do not include any of the extensions that are recently added. 
These extensions are only available when you use Cocoapods or manually copy them to your project.

**Comment by mhillebrand** [comment]
Thanks for the prompt action! Let me try again.

**Comment by mhillebrand** [comment]
I got the same error, even after deleting all things Carthage (including its cache in ~/Library/Caches/org.carthage.CarthageKit):

```
mhillebrand$ carthage update --platform iOS
*** Cloning Willow
*** Cloning PureLayout
*** Cloning EVReflection
*** Cloning AlamofireImage
*** Cloning Alamofire
*** Checking out Alamofire at "4.3.0"
*** Checking out AlamofireImage at "3.2.0"
*** Checking out EVReflection at "4.0.10"
*** Checking out PureLayout at "v3.0.2"
*** Checking out Willow at "3.0.0"
*** xcodebuild output can be found in /var/folders/ml/d9hzjcrd3mv_3hj_3qqf__brkdn63y/T/carthage-xcodebuild.AtqL5P.log
*** Building scheme "Alamofire iOS" in Alamofire.xcworkspace
*** Skipped building EVReflection due to the error:
Dependency "EVReflection" has no shared framework schemes for any of the platforms: iOS

If you believe this to be an error, please file an issue with the maintainers at Optional("https://github.com/evermeer/EVReflection/issues/new")
```

**Comment by evermeer** [comment]
Indeed the frameworks were not set to shared. That should now be solved.
